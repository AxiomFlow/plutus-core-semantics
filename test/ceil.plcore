(define result (builtin ceil (2.5)))
