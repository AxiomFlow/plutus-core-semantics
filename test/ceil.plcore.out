<T> <k> . </k> <lenv> .Map </lenv> <genv> ( Foo . result ) |-> 3e+00 </genv> <mod> Foo </mod> <sym> .Map </sym> </T>
